### PR TITLE
Azhidkov/multiple sources

### PR DESCRIPTION
**Pull request by d-r-q**

### CODE REVIEW

File: src/main/kotlin/jeb/Backuper.kt [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Backuper(private val storage: Storage, private val now: LocalDateTime) {
         log.info("Next tape number = $nextTapeNum")
 
         val lastBackup = storage.lastModified(File(state.backupsDir), { isTape(it, state) })
-        val fromDir = File(state.source)
+        val fromDir = state.source
```
Comment (aleksey-zhidkov) [inline src/main/kotlin/jeb/Backuper.kt]: Test review comment

File: src/main/kotlin/jeb/jeb.kt [hunk 2]
```diff
@@ -52,6 +53,21 @@ private fun doBackup(config: File, state: State, time: LocalDateTime) {
     }
 }
 
+private fun readSourceDirs(currentDir: String): List<String> {
+    val sources = arrayListOf<String>()
+    var defaultSourceDir: String? = currentDir
+    do {
+        val invitation = if (sources.size == 0) "Source directory (add trailing slash to backup directory content or leave it blank to backup directory)"
```
Comment (aleksey-zhidkov) [inline src/main/kotlin/jeb/jeb.kt]: Message too long, split it to two lines

**Review by d-r-q — COMMENTED** [review]
Fixes required

File: src/main/kotlin/jeb/jeb.kt (hunk 2 above)
Comment (aleksey-zhidkov) [inline src/main/kotlin/jeb/jeb.kt]: Done

**Pull request closed without merging**